From f736be156f8f1785b4c8bbfc27b7fd4761ca1656 Mon Sep 17 00:00:00 2001
From: Jian He <jianhe@apache.org>
Date: Mon, 2 Mar 2015 14:39:49 -0800
Subject: [PATCH 1387/2848] YARN-3281. Added RMStateStore to StateMachine
 visualization list. Contributed by Chengbing Liu
 (cherry picked from commit
 5d0bae550f5b9a6005aa1d373cfe1ec80513dbd9)

(cherry picked from commit 76b7283fa18641ce6e6c5ab5a774fdf0aa4564df)

Change-Id: I6ac2fdf9e9a2412febd41a4e0a38ec0d346c2954
---
 .../hadoop-yarn-server-resourcemanager/pom.xml     |    7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/pom.xml
index 92ec3c7..6262142 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/pom.xml
@@ -296,7 +296,7 @@
               <source>
                 <directory>${basedir}/src/main/proto</directory>
                 <includes>
-		          <include>yarn_server_resourcemanager_recovery.proto</include>
+                  <include>yarn_server_resourcemanager_recovery.proto</include>
                 </includes>
               </source>
               <output>${project.build.directory}/generated-sources/java</output>
@@ -326,10 +326,11 @@
                 </goals>
                 <configuration>
                   <mainClass>org.apache.hadoop.yarn.state.VisualizeStateMachine</mainClass>
-		  <classpathScope>compile</classpathScope>
+                  <classpathScope>compile</classpathScope>
                   <arguments>
                     <argument>ResourceManager</argument>
-                    <argument>org.apache.hadoop.yarn.server.resourcemanager.rmapp.attempt.RMAppAttemptImpl,
+                    <argument>org.apache.hadoop.yarn.server.resourcemanager.recovery.RMStateStore,
+                      org.apache.hadoop.yarn.server.resourcemanager.rmapp.attempt.RMAppAttemptImpl,
                       org.apache.hadoop.yarn.server.resourcemanager.rmapp.RMAppImpl,
                       org.apache.hadoop.yarn.server.resourcemanager.rmcontainer.RMContainerImpl,
                       org.apache.hadoop.yarn.server.resourcemanager.rmnode.RMNodeImpl</argument>
-- 
1.7.9.5

